pages/report-hazard/style.css

```diff
@@ -126,8 +125,7 @@ input[type='radio'] {
   background: var(--neutral-50);
 }
 
-#hazard-category input[type='radio']:checked + label,
-#hazard-option-content input[type='radio']:checked + label {
+#hazard-category input[type='radio']:checked + label {
   border-radius: 8px;
   border: 1px solid var(--neutral-100);
   background: var(--secondary-500);
```

Comment:
Add position: relative here to make the orange icon stick to the option UI card

```suggestion
  background: var(--secondary-500);
  position: relative;
```

Reply:
![image](https://github.com/whamester/WMDD-4885-Project/assets/87240930/408142be-f9c7-4bfe-a1d0-d930a2e87257)

Reply:
Better now

Reply:
perfect!